ui/app/components/pending-tx.js

```diff
@@ -15,9 +15,9 @@ const addressSummary = util.addressSummary
 const nameForAddress = require('../../lib/contract-namer')
 const BNInput = require('./bn-as-decimal-input')
 
-const MIN_GAS_PRICE_GWEI_BN = new BN(1)
-const GWEI_FACTOR = new BN(1e9)
-const MIN_GAS_PRICE_BN = MIN_GAS_PRICE_GWEI_BN.mul(GWEI_FACTOR)
+const MIN_GAS_PRICE_MWEI_BN = new BN('100')
+const MWEI_FACTOR = new BN(1e6)
+const MIN_GAS_PRICE_BN = MIN_GAS_PRICE_MWEI_BN.mul(MWEI_FACTOR)
```

Comment:
has the display changed to `MWEI`?

Reply:
This is only an internal change--the units reflected front-side are still in GWEI due to the scale/precision.

Reply:
if this is only internal, why not go all the way down to wei?

Reply:
We can--it was just a preference I suppose to use the closest metric unit. Can change.

Reply:
No, display is showing Gwei.